ursula_cli/shell.py

```diff
@@ -51,6 +52,9 @@ def _append_envvar(key, value):
     if key in os.environ:
         os.environ[key] = "%s %s" % (os.environ[key], value)
     else:
+        _set_envvar(key, value)
+
+def _set_envvar(key, value):
```

Comment:
nope,  I did at first and if I had the `SETTINGS_FILE=` environment already set then it appended it and got all mixed up.